Gemfile.lock

```diff
@@ -366,7 +366,7 @@ DEPENDENCIES
   mustermann (~> 0.4)
   mysql2 (~> 0.3.20)
   nifty-generators
-  paperclip (< 4.2.0)
+  paperclip (~> 4.2.2)
```

Comment:
Oops, could you revert this change please? Thank you!!

Reply:
Ah I didn't mean to add this, maybe we can set the specific version in Gemfile?